### DISCUSSION TITLE
add eclipse community installer

### DISCUSSION THREAD

**Discussion by Minetheground** [microsoft/winget-pkgs#22534] [Ideas] [answered]
can you add this because a lot of coders us this suite of programs like people who make mods for minecraft and java programs

**Comment by ImJoakim** [comment, marked as answer]
Hey there @Minethegroud, can you please create this under https://github.com/microsoft/winget-pkgs/issues?

The direct link for a package request/submission can be found here: [Issue: Package Request/Submission](https://github.com/microsoft/winget-pkgs/issues/new?assignees=&labels=Help-Wanted%2C+Package-Request&template=Manifest_Request.md&title=)
